util/lua/lua.go

```diff
@@ -151,6 +161,14 @@ func (vm VM) GetHealthScript(obj *unstructured.Unstructured) (string, bool, erro
 	// if not found in the ResourceOverrides at all, search it as is in the built-in scripts
 	// (as built-in scripts are files in folders, named after the GVK, currently there is no wildcard support for them)
 	builtInScript, err := vm.getPredefinedLuaScripts(key, healthScriptFile)
+	if err != nil {
+		var doesNotExist *ScriptDoesNotExistError
+		if errors.As(err, &doesNotExist) {
+			// It's okay if no built-in health script exists. Just return an empty string and let the caller handle it.
+			return "", false, nil
```

Comment:
```suggestion
			return "", false, nil
```
why are we returning error as nil here?